packages/engine/Specs/Scene/Vector3DTileContentSpec.js

```diff
@@ -329,70 +329,71 @@ describe(
         });
       });
 
-      it("styles points for color", () => {
-        return Cesium3DTilesTester.loadTileset(
+      it("styles points for color", async function () {
+        const tileset = await Cesium3DTilesTester.loadTileset(
```

Comment:
Can you clarify what changed here as well? :pray:

Reply:
Again, this was transitioning promise chains to async/await.